soc/arm/nordic_nrf/nrf91/soc.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file SoC configuration macros for the Nordic Semiconductor nRF91 family processors.
+ */
+
+#ifndef _NORDICSEMI_NRF91_SOC_H_
+#define _NORDICSEMI_NRF91_SOC_H_
+
+#ifndef _ASMLANGUAGE
+
+#include <nrf_common.h>
```

Comment:
This is needless.

Reply:
Removed.
Again, note that this inclusion is present in nrf51, and 52 soc.h headers.